skypy/galaxies/tests/test_redshift.py

```diff
@@ -32,7 +32,7 @@ def test_schechter_lf_redshift():
     x_min = 10.**(-0.4*(M_lim - M_star))
 
     # density with factor from upper incomplete gamma function
-    density = phi_star*gamma(alpha+1)*gammaincc(alpha+1, x_min)
+    density = phi_star*gamma(alpha+1)*gammaincc(alpha+1, x_min)*np.power(cosmo.h, 3)
```

Comment:
I don't think this is the right place to account for `h`. The parameter `phi_star` has the same dimension [unit] as the luminosity function, which is 1/V/luminosity [commonly Mpc^-3 mag^-1]. That means  `phi_star` must agree with these dimensions/units.

In your example, you are using the parameters of Tortorelli2020. In this work, they are giving `phi_star` in units of h_70 = H_0/70. So the input parameter must be corrected by this unit. However, as your defined cosmology includes H_0 = 70, h_70 is equal to 1. Therefore, the number that you have in your config files corresponds to units of Mpc^-3 mag^-1, which are as I said the physical units.

If we, however, want to enable `phi_star` to be given in units of h_0, we would also need to specify this in the documentation. But in this case, the parameter in your config file has the wrong value.